test/integration/config/sync-register-base-service-config-test.js

```diff
@@ -74,7 +74,7 @@ test('should load default localhost config file when calling "load"', t => {
 
 test('should load overrides file and inject env vars when calling "load"', t => {
   const config = new BaseServiceConfig()
-  process.env.SERVICE_ENVIRONMENT = 'production-env'
+  process.env.SERVICE_ENVIRONMENT = 'production'
```

Comment:
See above comments about `production-env`. It's really annoying that ava doesn't allow nesting... I'm afraid these tests are going to get out of sync. Async... out of async. Hah. Hah. But no, really. Is there not some way to avoid complete duplication with ava?